copycheck.py

```diff
@@ -2,6 +2,6 @@
 from shutil import copyfile
 
 if (os.environ.get('ZAPD_COPYDIR') != None):
```

Comment:
`ZAPD_COPYDIR` might need to be renamed to `ZAPD_OOTCOPYDIR` with an accompanying `ZAPD_MMCOPYDIR`. But that'll be for another time.